tools/.golangci.yaml

```diff
@@ -87,9 +87,12 @@ linters-settings: # please keep this alphabetized
       - name: error-strings
         severity: error
         disabled: false
-      # TODO: enable the following rules
       - name: var-naming
-        disabled: true
+        disabled: false
+        arguments:
+          - [] # AllowList: leave it empty to go with defaults, not relaxing the rule.
+          - ["GRPC", "WAL"] # DenyList: Add GRPC and WAL to strict the rule not allowing instances like Wal or Grpc.
```

Comment:
We already added some skipping directive in the repo for some specific cases. Can we just remove both list so that we just use the default settings?

Reply:
> We already added some skipping directive in the repo for some specific cases.

Is there another place in which we have skipped directives?

> Can we just remove both list so that we just use the default settings?

As I was working on enabling the rule I found inconsistent use of WAL/Wal and GRPC/Grpc. If we decide to remove it from the deny list, the linter won't complain about the mixed usage of these keywords.

Reply:
> > We already added some skipping directive in the repo for some specific cases.
> 
> Is there another place in which we have skipped directives?

Ah, got it. Yes, we have them inline in the code. However, new instances (i.e., new variables, new struct fields), won't be evaluated against this rule, so it will allow new instances of Wal and Grpc.

Reply:
I am curious what's the default settings/values for the allowlist and blocklist? If we leave both slices as empty, does it have the same effect as disabling the linter?

Reply:
The AllowList is empty as far as I know. The DenyList has the following items: https://github.com/mgechev/revive/blob/v1.3.7/lint/utils.go#L93-L133

I had to dive into the code to get these values, as they're not publicly documented (it's an unexported variable). As you can see, it doesn't include GRCP, and (of course) WAL.

Reply:
Thanks for the clarification. Can you please add a comment to link to it so that all contributors are well aware of the. default values for blockList?

Reply:
Done. PTAL :)